protocol-units/bridge/indexer-db/src/client.rs

```diff
@@ -185,36 +85,70 @@ impl Client {
 			.filter(initiated_events::bridge_transfer_id.eq(bridge_transfer_id.clone()))
 			.load::<InitiatedEvent>(&mut self.conn)?;
 
-		let locked_events = locked_events::table
-			.filter(locked_events::bridge_transfer_id.eq(bridge_transfer_id.clone()))
-			.load::<LockedEvent>(&mut self.conn)?;
-
-		let initiator_completed_events = initiator_completed_events::table
-			.filter(initiator_completed_events::bridge_transfer_id.eq(bridge_transfer_id.clone()))
-			.load::<InitiatorCompletedEvent>(&mut self.conn)?;
+		let completed_events = completed_events::table
+			.filter(completed_events::bridge_transfer_id.eq(bridge_transfer_id.clone()))
+			.load::<CompletedEvent>(&mut self.conn)?;
 
-		let counter_party_completed_events = counter_party_completed_events::table
-			.filter(
-				counter_party_completed_events::bridge_transfer_id.eq(bridge_transfer_id.clone()),
-			)
-			.load::<CounterPartyCompletedEvent>(&mut self.conn)?;
-
-		let cancelled_events = cancelled_events::table
-			.filter(cancelled_events::bridge_transfer_id.eq(bridge_transfer_id.clone()))
-			.load::<CancelledEvent>(&mut self.conn)?;
+		Ok(BridgeEventPackage { initiated_events, completed_events })
+	}
 
-		let refunded_events = refunded_events::table
-			.filter(refunded_events::bridge_transfer_id.eq(bridge_transfer_id.clone()))
-			.load::<RefundedEvent>(&mut self.conn)?;
+	/// Inserts a new transfer action into the database.
+	pub fn insert_transfer_action(
```

Comment:
I thought we were removing this @musitdev ?

Reply:
This is the `Indexer-Relayer Mode` from the discussion #884. Effectively there's not much need, only debugging purpose I think.